BowOpenAPI/BowOpenAPICommand.swift

```diff
@@ -0,0 +1,17 @@
+//  Copyright © 2019 The Bow Authors.
+
+import ArgumentParser
+
+struct BowOpenAPICommand: ParsableCommand {
+    static var configuration = CommandConfiguration(commandName: "bow-openapi",
+                                                    abstract: "Generate a Swift network client from an OpenAPI / Swagger specification file." )
+    
+    @Option(help: "The name for the output module.")
```

Comment:
```suggestion
    @Argument(help: "The name for the output module.")
```
This parameter is required